src/compass/s1_geocode_slc.py

```diff
@@ -243,7 +245,8 @@ def run(cfg: GeoRunConfig):
                 corrections_to_h5group(correction_group, burst, cfg, rg_lut, az_lut,
                                        scratch_path,
                                        weather_model_path=cfg.weather_model_file,
-                                       delay_type=cfg.tropo_params.delay_type)
+                                       delay_type=cfg.tropo_params.delay_type,
+                                       flag_antenna_pattern_by_sas=flag_antenna_pattern_by_sas)
```

Comment:
```suggestion
                                       flag_antenna_pattern_by_sas=check_eap.phase_correction:)
```
More direct and can remove flag variable above.